packages/api/src/controllers/stream.ts

```diff
@@ -1364,8 +1374,8 @@ app.post(
 
     await triggerCatalystStreamUpdated(req, stream.playbackId);
 
-    res.status(204);
-    res.end();
+    res.status(200);
+    res.json(newTarget);
```

Comment:
You should call `db.multistreamTarget.cleanWriteOnlyResponse` [(e.g.)](https://github.com/livepeer/livepeer-com/blob/37ab514c365bf0ccae3f787dc6afb8a13785fe2b/packages/api/src/controllers/multistream.ts#L85) on this object before returning it on the response payload. It will clean up the private URL field (and maybe some other), which is not supposed to go in API responses.

It's theoretically not a huge issue here, since the user just sent the URL on the request, but I think we should keep it consistent (and eventually there could be other secret fields).

Reply:
Updated. Thanks for spotting that!